Header header
# mass of tool (i.e. iiwa param m7)
float64 m
# center of gravity of tool (i.e. iiwa param sp7z)
float64 spx
float64 spy
float64 spz
# float64 tool_Ixx
# float64 tool_Iyy
# float64 tool_Izz
